# mirrored from AtlasControlTypes.h
uint32 status_flags    # use AtlasBeahviorFeedback/status_flags
float64 t_step_rem
uint32 current_step_index
uint32 next_step_index_needed
atlas_msgs/AtlasBehaviorStepData desired_step_saturated
